samples/snippets/csharp/VS_Snippets_CFX/c_standalonedatacontractserializer/cs/source.cs

```diff
@@ -42,19 +42,19 @@ public class LibraryPatron
         [DataContract]
         public class LibraryItem
         {
-            //code not shown�
+            //code not shown�
```

Comment:
What is the final character (rendered as a question mark in a triangle) supposed to be? Make sure the file is saved using UTF-8 encoding (it's probably saved using ASCII encoding).